assets/src/data/eventespresso/core/resolvers/relations.js

```diff
@@ -250,18 +249,17 @@ export function* getRelatedEntitiesForIds(
 	if ( ! response.length ) {
 		return DEFAULT_EMPTY_ARRAY;
 	}
-	const relationPrimaryKey = getPrimaryKey(
-		singularModelName( relationName )
-	);
-	const modelPrimaryKey = getPrimaryKey( singularModelName( modelName ) );
+	const relationPrimaryKey = getPrimaryKey( relationName );
+	const modelPrimaryKey = getPrimaryKey( modelName );
+	const pluralRelationName = pluralModelName( relationName );
```

Comment:
ok this looks very intentional

Reply:
Yes, a relation record may be in the response in its plural form.  This is an example where the rest api is "inconsistent" (in quotes, because from a certain perspective this might actually be seen as consistent).  Relation records might be singular or plural depending on they type of relation.